app/src/server/api/routers/projects.router.ts

```diff
@@ -258,6 +258,14 @@ export const projectsRouter = createTRPCRouter({
             lastViewedProjectId: newProjectId,
           },
         }),
+        prisma.creditAdjustment.create({
```

Comment:
Actually let's only do this for the first default project we create when a user signs up, instead of for every new project an existing user creates.

Reply:
Sure, changed it!